third_party/3/pkg_resources/py31compat.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import Text
+import os
+import sys
+
+needs_makedirs: bool
+
+def _makedirs_31(path: Text, exist_ok: bool = ...) -> None: ...
+
+# _makedirs_31 has special behavior to handle an edge case that was removed in
+# 3.4.1. No one should be using 3.4 instead of 3.4.1, so this should be fine.
+if sys.version_info >= (3, 4):
```

Comment:
`py31compat.py` has:
```
needs_makedirs = (
    six.PY2 or
    (3, 4) <= sys.version_info < (3, 4, 1)
)
makedirs = _makedirs_31 if needs_makedirs else os.makedirs
```
(https://github.com/pypa/setuptools/blob/master/pkg_resources/py31compat.py)
so `_makedirs_31` is used in Python 2 and Python 3.4.0. If we go with your assumption that we can ignore 3.4.0 (which I think is fine), then the version check should be `>= (3,)`.

Reply:
Done.